<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RobotControlClass</class>
 <widget class="QMainWindow" name="RobotControlClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1335</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RobotControl</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="CQtOpenCVViewerGl" name="openCVviewer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>1200</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>690</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Client IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>181</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="mBox">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>630</y>
      <width>601</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>610</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>1240</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CQtOpenCVViewerGl</class>
   <extends>QOpenGLWidget</extends>
   <header>cqtopencvviewergl.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="robotcontrol.qrc"/>
 </resources>
 <connections/>
</ui>
